ci.rb

```diff
@@ -297,7 +298,7 @@ def get_limits(t_options, t_client, t_repo)
                 # now build this branch
                 if File.directory?(p.get_src_dir)
                   $logger.info "Removing pre-existing branch directory (#{p.get_build_dir})"
-                  system("rm -rf #{p.get_src_dir}")
+                  FileUtils.rm_rf(p.get_src_dir)
```

Comment:
Seems to work great now that we are closing test.xml when we are done with it.  Fixes #12